files/vue/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "vue",
+  "repo": "yyx990803/vue",
+  "files": {
+	"basePath": "dist/",
+    "include": ["vue.min.js"]
```

Comment:
Because we already host the unminified file http://www.jsdelivr.com/#!vue please add it here as well.